strongbox-web-core/src/test/resources/strongbox-authentication-providers.xml

```diff
@@ -1,23 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns:context="http://www.springframework.org/schema/context"
-       xmlns:util="http://www.springframework.org/schema/util"
-       xmlns:cache="http://www.springframework.org/schema/cache"
-       xmlns:security="http://www.springframework.org/schema/security"
-       xmlns="http://www.springframework.org/schema/beans"
-       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+    xmlns:context="http://www.springframework.org/schema/context"
+    xmlns:util="http://www.springframework.org/schema/util"
+    xmlns:cache="http://www.springframework.org/schema/cache"
+    xmlns:security="http://www.springframework.org/schema/security"
+
+    xmlns="http://www.springframework.org/schema/beans"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
                            http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
                            http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache.xsd
                            http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
                            http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
                            http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd">
 
     <context:annotation-config />
+    <cache:annotation-driven />
 
     <bean id="xmlUserDetailService" class="org.carlspring.strongbox.users.userdetails.XmlUserDetailService"/>
 
     <bean id="authenticationProviderFirst" class="org.carlspring.strongbox.controllers.configuration.security.authentication.AuthenticatorsConfigControllerTestIT.TestAuthenticationProvider"/>
     <bean id="authenticationProviderSecond" class="org.carlspring.strongbox.controllers.configuration.security.authentication.AuthenticatorsConfigControllerTestIT.TestAuthenticationProvider"/>
     <bean id="authenticationProviderThird" class="org.carlspring.strongbox.controllers.configuration.security.authentication.AuthenticatorsConfigControllerTestIT.TestAuthenticationProvider"/>
 
-</beans>
+    <bean
```

Comment:
@carlspring @fuss86 it's seems we don't need this changes

Reply:
@fuss86 / @sbespalov ,

Can we not drop this XML...?

Reply:
I'll find out, I don't remember, really :(

Reply:
> Can we not drop this XML...?

no, we can't. I mean that this XML used only for specific tests and we shouldn't change or drop it or even use for all tests.